src/main.cpp

```diff
@@ -136,6 +136,7 @@ void usage()
   std::cerr << "    BPFTRACE_MAX_PROBES               [default: 512] max number of probes" << std::endl;
   std::cerr << "    BPFTRACE_MAX_STRLEN               [default: 64] bytes on BPF stack per str()" << std::endl;
   std::cerr << "    BPFTRACE_MAX_TYPE_RES_ITERATIONS  [default: 0] number of levels of nested field accesses for tracepoint args" << std::endl;
+  std::cerr << "    BPFTRACE_ON_STACK_LIMIT           [default: 32] max bytes of individual objects to store them on the BPF stack vs. pre-allocated memory" << std::endl;
```

Comment:
I don't think this needs to be an environment variable as well as a config option. It seems unlikely that someone would want to change this setting globally for all scripts on their system - it's a thing that'd be very specific to a single script. (this is true for a lot of our env vars, but they exist as historical baggage since they were around before config options were a thing)

Reply:
ack will remove env var